### ISSUE TITLE
Does not handle intrinsic modules

### ISSUE THREAD

**Issue by Beliavsky** [lycantropos/run-fortran#2]
For a program that uses an intrinsic module such as iso_fortran_env, for example

```
use iso_fortran_env
print*,"input unit =",input_unit
end
```
The run-fortran script looks for the definition of iso_fortran_env in a source file and crashes (output below) when it cannot find it. Ideally it would ignore statements using intrinsic modules such as iso_fortran_env and iso_c_binding

```
Traceback (most recent call last):
  File "c:\python3\code\public_domain\run-fortran\run-fortran.py", line 174, in get_module_path_by_module_name
    res, = modules_paths
ValueError: not enough values to unpack (expected 1, got 0)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\python3\code\public_domain\run-fortran\run-fortran.py", line 292, in <module>
    main()
  File "c:\intelpython3\lib\site-packages\click\core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "c:\intelpython3\lib\site-packages\click\core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "c:\intelpython3\lib\site-packages\click\core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\intelpython3\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\intelpython3\lib\site-packages\click\core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "c:\python3\code\public_domain\run-fortran\run-fortran.py", line 82, in run
    update_chained_modules_names(modules_names_by_modules_paths)
  File "c:\python3\code\public_domain\run-fortran\run-fortran.py", line 202, in update_chained_modules_names
    get_module_path_by_module_name(used_module_name)]
  File "c:\python3\code\public_domain\run-fortran\run-fortran.py", line 192, in get_module_path_by_module_name
    raise ValueError(err_msg) from err
ValueError: Requested module name is not found: no appearance of module named "iso_fortran_env" in modules definitions.


```

**Comment by lycantropos** [comment]
Can you please explain to me how can we tell whether the given module is "intrinsic"? Does it depend on the compiler/platform/standard used?

What is the potential solution: add possibility for a user to specify list of such modules?

**Comment by Beliavsky** [comment]
There are [five intrinsic modules](https://riptutorial.com/fortran/example/5650/intrinsic-modules) that are part of the Fortran 2003 standard:

ISO_C_Binding
ISO_Fortran_env
IEEE_Exceptions
IEEE_Arithmetic
IEEE_Features

The Python code could be told to ignore these modules when figuring out dependencies.

**Comment by lycantropos** [comment]
Hi, I've updated script to add support for intrinsic modules
https://pypi.org/project/run-fortran/0.2.0/
can you please try and test it for your project?

**Comment by lycantropos** [comment]
Also I've released [`1.0.0` version](https://pypi.org/project/run-fortran/1.0.0/) that changes command from
```bash
run-fortran run [-p path]
```
to
```bash
run-fortran path [rest paths...]
```
